apps/web/src/components/invoice-generator/index.tsx

```diff
@@ -253,31 +265,37 @@ function InvoiceItem({
           required
         />
       </ErrorMessageBlock>
-      <ErrorMessageBlock
-        elementName={`rows[${index.toString()}].quantity`}
-        formState={state}
-      >
-        <InputRow
-          label={t("Quantity")}
-          name="rows.quantity"
-          id={`rows[${index.toString()}].quantity`}
-          type="number"
-          required
-        />
-      </ErrorMessageBlock>
-      <ErrorMessageBlock
-        elementName={`rows[${index.toString()}].unit`}
-        formState={state}
-      >
-        <InputRow
-          label={t("Unit")}
-          name="rows.unit"
-          id={`rows[${index.toString()}].unit`}
-          defaultValue="kpl"
-          maxLength={128}
-          required
-        />
-      </ErrorMessageBlock>
+      <span className="flex justify-center">
```

Comment:
There might be a more semantic element to use here to group two inputs, instead of `<span>`, perhaps [`<fieldset>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/fieldset)? Wdyt?

Reply:
Yes I think that a <fieldset> fits better here. Thank you for the suggestion!